q2_fondue/scraper.py

```diff
@@ -295,16 +288,17 @@ def scrape_collection(
     """
     logger.setLevel(log_level.upper())
 
+    dotenv.load_dotenv()
+
     logger.info(
-        f'Scraping accession IDs for collection "{collection_name}" in '
-        f'{library_type} library with user ID {user_id}...'
+        f'Scraping accession IDs for collection "{collection_name}"...'
     )
 
     # initialise Zotero instance
     zot = zotero.Zotero(
-        user_id,
-        library_type,
-        api_key)
+        os.getenv('USERID'),
+        os.getenv('TYPE'),
+        os.getenv('APIKEY'))
```

Comment:
I would suggest adding a prefix to all of those vars (something like `ZOTERO_`) to be a bit more explicit and avoid potential clashes with variables already existing in user's environment.